website/servers/index.php

```diff
@@ -1,6 +1,7 @@
 <?php
 
 include_once 'servers.lib.php';
+include_once __DIR__ . '/../../lib/ntbb-database.lib.php';
```

Comment:
What is this for? `servers` already includes `session` which includes `database`.

Reply:
```suggestion
```